news/_posts/core-weekly/2020-25-05-coreweekly-20-2020.md

```diff
@@ -0,0 +1,142 @@
+* [#19137](https://github.com/PrestaShop/PrestaShop/pull/19137): Functional tests - Sort order messages table. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#19133](https://github.com/PrestaShop/PrestaShop/pull/19133): Functional tests - Delete boBasePage for functional tests orders, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#19127](https://github.com/PrestaShop/PrestaShop/pull/19127): Functional tests - Delete bo base page from sanity tests , by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#19126](https://github.com/PrestaShop/PrestaShop/pull/19126): Functional tests - Sort all tables in monitoring page. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#19109](https://github.com/PrestaShop/PrestaShop/pull/19109): Functional tests - Add test subscribe to newsletter from BO, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#19092](https://github.com/PrestaShop/PrestaShop/pull/19092): Functional tests - Edit home category. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#19088](https://github.com/PrestaShop/PrestaShop/pull/19088): Functional tests - Filter and sort files. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#19074](https://github.com/PrestaShop/PrestaShop/pull/19074): Functional tests - Add test pagination order message, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#19071](https://github.com/PrestaShop/PrestaShop/pull/19071): Functional tests - Add test taxes pagination, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#19069](https://github.com/PrestaShop/PrestaShop/pull/19069): Functional tests - Profiles pagination. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#19067](https://github.com/PrestaShop/PrestaShop/pull/19067): Functional tests - Sort profiles. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#19066](https://github.com/PrestaShop/PrestaShop/pull/19066): Functional test - Add test quick edit languages, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#19027](https://github.com/PrestaShop/PrestaShop/pull/19027): Functional tests - Export sql query. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+
+
+## Code changes in modules, themes & tools
+
+
+### Changes in developer documentation
+* [#540](https://github.com/PrestaShop/docs/pull/540): Add a validation rule to avoid collisions between modules , by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#539](https://github.com/PrestaShop/docs/pull/539): Update dependencies for security reasons, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#536](https://github.com/PrestaShop/docs/pull/536): Add info about the CQRS debug toolbar, by [@jolelievre](https://github.com/jolelievre)
+* [#535](https://github.com/PrestaShop/docs/pull/535): Update _index.md. Thank you [@derekjwhitten](https://github.com/derekjwhitten)
```

Comment:
```suggestion
* [#535](https://github.com/PrestaShop/docs/pull/535): Update index.md. Thank you [@derekjwhitten](https://github.com/derekjwhitten)
```

Reply:
will you edit the pull request title if I change it in the core weekly?

Reply:
Euh no ! It's the file name ;) `_index.md`

Reply:
so why is there a _ in the pr title?

Reply:
Because it is in the file path.

Reply:
I  still don't get it, but let's commit this and merge :-)